versions/v0.9.1/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:4
+
+MAINTAINER Christopher Biscardi <chris@christopherbiscardi.com>
+
+WORKDIR /tmp
```

Comment:
It doesn't look like this is used in the build instructions, we should be good to get rid of it. `/tmp` is a bit of a weird default for users of this image.